.github/workflows/ci.yml

```
@@ -72,3 +72,15 @@ jobs:
 
       - name: Run Integration Tests
         run: ./gradlew integrationTest
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
```

actionlint (1.7.7)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)